### ISSUE TITLE
Acme Thread fails

### ISSUE THREAD

**Issue by gumyr** [gumyr/bd_warehouse#32]
`  acme = AcmeThread(size="1/4", length=1 * IN)` results in:

  File "/home/nathaniel/projects/gardenia/.venv/lib/python3.12/site-packages/bd_warehouse/thread.py", line 743, in __init__
    super().__init__(
  File "/home/nathaniel/projects/gardenia/.venv/lib/python3.12/site-packages/bd_warehouse/thread.py", line 605, in __init__
    self.lead = pitch * starts
                ~~~~~~^~~~~~~~
TypeError: can't multiply sequence by non-int of type 'float'

**Comment by gumyr** [comment]
The problem is that the `TrapezoidalThread` class has a new `starts` parameter which isn't being set correctly.

**Comment by gumyr** [comment]
Fixed with commit 53a69f7ba925596ed8d627e7e38cd49f8ff03ffb
